lib/handleRequest.js

```diff
@@ -7,10 +7,7 @@ const wrapThenable = require('./wrapThenable')
 
 function handleRequest (err, request, reply) {
   if (reply.sent === true) return
-  if (err != null) {
-    reply.send(err)
-    return
-  }
+  if (err != null) return reply.send(err)
```

Comment:
the previous behavior was different, and there for a reason ;).

Reply:
I see now, sorry about that changed without thinking much 😄